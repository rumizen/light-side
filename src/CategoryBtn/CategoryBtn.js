import React from "react";
import "./CategoryBtn.scss";

const CategoryBtn = ({ name, icon }) => {
  return (
    <button className="category-btn">
      <h3>{name}</h3>
      {icon}
    </button>
  );
};

export default CategoryBtn;
