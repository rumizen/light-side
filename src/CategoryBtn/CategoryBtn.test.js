import React from "react";
import { shallow, mount } from "enzyme";
import CategoryBtn from "./CategoryBtn";

describe("CategoryBtn", () => {
  it("should match the CategoryBtn snapshot", () => {
    const wrapper = shallow(
      <CategoryBtn
        name="People"
        icon="ðŸ‘¥"
      />);

    expect(wrapper).toMatchSnapshot();
  });
});
