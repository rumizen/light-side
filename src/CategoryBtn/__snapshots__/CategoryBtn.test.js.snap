// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`CategoryBtn should match the CategoryBtn snapshot 1`] = `ShallowWrapper {}`;
